qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -139,38 +138,35 @@ def main(self):
         success = False
         try:
             os.makedirs(self.options.tmpdir, exist_ok=False)
```

Comment:
Good idea. I needed to put start_logging() below the os.makedirs() call, but you're right that this should be outside the large try/except block. I've now moved them both up.